MediaBrowser.Controller/MediaBrowser.Controller.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="3.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.1" />
```

Comment:
Required for `IConfiguration.GetValue<bool>()` used here: https://github.com/jellyfin/jellyfin/pull/2601/files#diff-c54ff5039ee8fbb6c22dcbd1819ef89bR34